commit 1edad95e65a73331264d50a7c294182c6c04cbcf (HEAD -> master, refs/changes/48/5248/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2019-01-23T12:04:05-08:00 (9 months ago)
    
    MANTA-4035 retire propeller
    Reviewed by: Kody A Kantor <kody@kkantor.com>
    Approved by: Kody A Kantor <kody@kkantor.com>

diff --git a/update/repos.sh b/update/repos.sh
index 8281754..85079ab 100755
--- a/update/repos.sh
+++ b/update/repos.sh
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2017, Joyent, Inc.
+# Copyright (c) 2019, Joyent, Inc.
 #
 
 ###############################################################################
@@ -33,7 +33,6 @@ REPOS=(
     manta-medusa
     manta-mola
     manta-muskie
-    manta-propeller
     manta-reshard
     manta-wrasse
     moray
